### PR TITLE
Tooltips2

### PR DESCRIPTION
**Pull request by ppwadhwa**
# Description

This PR adds in a few more key bindings to tool tips.  

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: napari/_qt/layer_controls/qt_labels_controls.py [hunk 1]
```diff
@@ -155,12 +153,11 @@ def __init__(self, layer):
         self.paint_button = QtModeRadioButton(
             layer, 'paint', Mode.PAINT, tooltip='Paint mode (P)'
         )
-        btn = 'Cmd' if sys.platform == 'darwin' else 'Ctrl'
         self.fill_button = QtModeRadioButton(
-            layer, 'fill', Mode.FILL, tooltip=f'Fill mode ({btn})'
+            layer, 'fill', Mode.FILL, tooltip='Fill mode (F)'
```
Comment (sofroniewn) [inline napari/_qt/layer_controls/qt_labels_controls.py]: So it's a little funny here, but we actually allow users to temporarily enable fill and erase using modifier keys, which is why these were documented as they were - see https://github.com/napari/napari/blob/e33c2fd1d9e3456ac7cfef664d25209c411c8da4/napari/layers/labels/_labels_key_bindings.py#L91-L116.

Personally I'm not such a fan of that behaviour as whenever you press the modifier you see the button toggle even if you're just using the modifier as a modifier. I'm curious @jni how you feel about this, and if you've been using this functionality.

I'm not quite sure how to document it either if we do stick with it ...

**Review by sofroniewn — APPROVED** [review]
Looks great to me! Will be much nicer when this stuff is autogenerated one day, but thanks for manually adding it. I left one comment about some weirdness we currently have around using modifiers to temporarily enable button modes. I'm not sure it should hold up this PR, but wanted to flag it. Maybe @jni can weigh in quickly. It does feel odd to me. If we need additional opinions we can reach out to more people too

File: napari/_qt/layer_controls/qt_labels_controls.py (hunk 1 above)
Comment (jni) [reply napari/_qt/layer_controls/qt_labels_controls.py]: Personally I like the current behaviour, though I haven't developed the muscle memory for it. I can see how it would be very useful if someone masters it, though, so I'd like to keep it, and somehow document it better...

Comment (ppwadhwa) [reply napari/_qt/layer_controls/qt_labels_controls.py]: thanks for looking!  Just to make sure I'm clear---you want to stick with using 'Control' and 'Alt' for the 2 keys that you can press to temporarily activate fill and erase?  I can certainly change it back.

Comment (ppwadhwa) [reply napari/_qt/layer_controls/qt_labels_controls.py]: OK, so just re-reading what you both said---  having better documentation around all of this is desired.  I saw that you can do 2 things with each action-- you can either use a key to activate the fill or erase buttons (F or E), or you can temporarily activate the buttons while Control or Alt are pressed.  I saw that both work, but which one would you want in the tooltip?  I used the activated options (F or E) because that was consistent with the rest of the buttons.  However, if temporarily activating a button is getting more use, I can put that in there.  so, now the concern is how to explain in a tooltip?  I can try to explain more in the tooltip...I'll try some options and you can let me know what you think.

Comment (sofroniewn) [reply napari/_qt/layer_controls/qt_labels_controls.py]: >  I can try to explain more in the tooltip...I'll try some options and you can let me know what you think.

Thanks! I think (F/E) are more important, but maybe a two line tooltip with the permanent on top, and the temporary below would be good.

```
Fill mode (F)
toggle with (alt)
```

I dunno though, that's just an idea. Curious to see what you come up with!

Comment (sofroniewn) [reply napari/_qt/layer_controls/qt_labels_controls.py]: This might make autogeneration really hard, but I think at one point @goanpeca or @Carreau did ask if we'd ever have multiple keys for the same action, and while this isn't quite that it is close!

Comment (Carreau) [reply napari/_qt/layer_controls/qt_labels_controls.py]: Hum, I hadn't consider the case where keeping it pressed temporarily set it. 
My guess is that can always have an escape hatch, that is to say "F" would be managed by the ActionManager and Alt with custom code/text in tooltip. I'm not too sure it is worth having the complexity (yet), if it is a rare occurence.

File: napari/_qt/layer_controls/qt_shapes_controls.py [hunk 2]
```diff
@@ -154,7 +168,7 @@ def __init__(self, layer):
             layer,
             'delete_shape',
             slot=self.layer.remove_selected,
-            tooltip='Delete selected shapes',
+            tooltip=f"Delete selected shapes ({KEY_SYMBOLS['Backspace']})",
```
Comment (Carreau) [inline napari/_qt/layer_controls/qt_shapes_controls.py:171]: side-note;  we seem to often have the logic to use KEY_SYMBOL... would it be worth having a utils function somewhere that format shortcuts ? There are often other differences in rendering, like windows tend to separate keys with -, like `Ctrl-Alt-Del`, while mac will just stick the symbols w/o separator `⌘⌥⌫`.

We can likely do that as a separate PR if that sounds ok.

Comment (sofroniewn) [reply napari/_qt/layer_controls/qt_shapes_controls.py:171]: yeah, that sounds like a fine thing for a follow up PR

**Review by Carreau — APPROVED** [review]

File: napari/_qt/layer_controls/qt_labels_controls.py (hunk 1 above)
Comment (ppwadhwa) [reply napari/_qt/layer_controls/qt_labels_controls.py]: I updated the tooltips.  I liked what you had suggested @sofroniewn and couldn't come up with something better (I tried).

**Review by sofroniewn — APPROVED** [review]
Ok nice, this looks good. Let's get it in, we can continue to improve!
